modules/hibernating-cluster-hibernate.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-hibernate_{context}"]
+= Hibernating a cluster
+
+You can hibernate a cluster for up to 90 days. The cluster can recover if certificates expire while the cluster was in hibernation.
+
+.Prerequisites
+
+* The cluster has been running for at least 24 hours to allow the first certificate rotation to complete.
++
+If you must hibernate your cluster before the 24 hour certificate rotation, then use the following procedure instead: link:https://www.redhat.com/en/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms[Enabling OpenShift 4 Clusters to Stop and Resume Cluster VMs].
+
+* You have access to the cluster as a user with the `cluster-admin` role.
++
+// TODO: Confirm must be a cluster-admin
+
+.Procedure
+
+. Take an etcd backup before hibernating your cluster. For instructions, see "Backing up etcd".
++
+// TODO: Do we need a warning like the graceful shutdown has? If something goes wrong, will they follow the steps to "restore to a previous cluster state"?
+
+. Confirm that your cluster has been installed for at least 24 hours.
++
+// TODO: Do we have an easy command for them to check this? Or check that the certs have been rotated?
```

Comment:
@vrutkovs Should we base this solely on the time of 24 hours, or is there a command that we could provide that they could easily check whether the first rotation has completed?

Reply:
Its time based, yes. We don't have a way to check if your cluster is eligible, but we have a way to force rotate certificates early - https://www.redhat.com/en/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms

Reply:
Great thanks, I'll leave it as 24 hours then